### DISCUSSION TITLE
Miracles and Prayers ideas

### DISCUSSION THREAD

**Discussion by Hoppelite** [Lambosaurus/hives-and-torches#28] [Mechanics]
So basically the idea is that a prayer will have 3 tenets that will relate to miracles or curses, and one passive ability
I am planning on miracles and prayers to be somewhat like this example:

---

### Tenets
**Heal those in need**: Grant Miracle [Restore Wound](#restore-wound)
**Protect the weak**: Grant Miracle [Shield of Faith](#shield-of-faith)
**End their suffering**: Grant Curse [Path to the Grave](#path-to-the-grave)

### Passive
Your shield of faith adds your [Will](#will) as a bonus to targets with no wounds

## Restore Wound
The target restores 1 wound

## Shield of Faith
The target gains a 1d4 bonus to their defence for the rest of combat

## Path to the Grave
For the rest of the combat, whenever the target takes fatigue damage, add your [Will](#will) as bonus damage

---

The idea is that the Miracles and Curses will be mixed and matched in prayers, and the passive kind of gives a bonus theme.

Thoughts?

**Comment by LouderthanThunder** [comment]
Would you make it so the player is bound to the tenets of their prayer?

**Reply by Hoppelite** [reply]
Hmm, that could be interesting. I wanted the tenets to be fairly vague, as the interpretation will probably be relevant to whatever their divine patron is. I guess your patrons `Domain` would define which prayers you have access to.
